### ISSUE TITLE
Any suggestions for tuning the training parameters?

### ISSUE THREAD

**Issue by NUAAZXY** [jbloomAus/SAELens#67]
Thank you for providing an open-source dictionary learning repository. I have trained on other models similar to GPT-2. Despite numerous parameter adjustments, the training results still exhibit a large number of dead features as shown below.
![11](https://github.com/jbloomAus/mats_sae_training/assets/104181523/fd674f7a-88db-447e-9dbc-7aed2b63bb80)

**Comment by jbloomAus** [comment]
Probably lowering your learning rate will help, but dead features are a
somewhat ubiquitous issue at the moment with various people working on
solutions. If you share your full hyperparameter set and dashboards on
wandb I may have more suggestions.

**Comment by NUAAZXY** [comment]
Lowering the learning rate helps, thanks!
